src/test/java/org/jabref/logic/importer/WebFetchersTest.java

```diff
@@ -62,7 +71,9 @@ public void getSearchBasedFetchersReturnsAllFetcherDerivingFromSearchBasedFetche
     public void getFullTextFetchersReturnsAllFetcherDerivingFromFullTextFetcher() throws Exception {
         List<FulltextFetcher> fullTextFetchers = WebFetchers.getFullTextFetchers(importFormatPreferences);
 
-        Set<Class<? extends FulltextFetcher>> expected = reflections.getSubTypesOf(FulltextFetcher.class);
+        Set<Class<? extends FulltextFetcher>> expected = Stream.of(
```

Comment:
The test was written to remind when somebody implements a new fetcher but forgets to add it in `getFullTextFetchers`. This purpose is defeated if the list of fetchers is specified explicitly here. What was the reason for the change?

Reply:
The problem was that the reflections find the classes from the comment before as sub classes then. Maybe mocking them does change this behavior!